tests/unit/models/360-core.js

```diff
@@ -165,6 +165,54 @@ describe("SerSol 360 Core Model >", function() {
     });
   });
 
+  describe("serSol360Core model showJournalCoverImages method >", function() {
+    beforeEach(function() {
+      delete browzine.journalCoverImagesEnabled;
+    });
+
+    afterEach(function() {
+      delete browzine.journalCoverImagesEnabled;
+    });
+
+    it("should show journal cover when configuration property is undefined or null", function() {
+      expect(serSol360Core.showJournalCoverImages()).toEqual(true);
+    });
+
+    it("should show journal cover when configuration property is true", function() {
+      browzine.journalCoverImagesEnabled = true;
+      expect(serSol360Core.showJournalCoverImages()).toEqual(true);
+    });
+
+    it("should not show journal cover when configuration property is false", function() {
+      browzine.journalCoverImagesEnabled = false;
+      expect(serSol360Core.showJournalCoverImages()).toEqual(false);
+    });
+  });
+
+  describe("serSol360Core model showJournalBrowZineWebLinkText method >", function() {
```

Comment:
Same with this - we check that `serSol360Core.showJournalBrowZineWebLinkText()` is true or false properly, but we don't test that `serSol360Core.showJournalBrowZineWebLinkText()` influences what appears in the DOM from what I can see.  If we add that test, we'll be really tight on our coverage though 👍